docker/docker-compose.yml

```diff
@@ -61,10 +61,14 @@ services:
       TRACE_TARGET: ${TRACE_TARGET}
       TRACE_MSG_PCT: ${TRACE_MSG_PCT}
       ACK_ERROR_PCT: ${ACK_ERROR_PCT}
+      OIDC_JWKS_URI: ${OIDC_JWKS_URI}
+      CONTROLLER_API_KEY: ${CONTROLLER_API_KEY}
```

Comment:
If this is set, shouldn't we *also* set the agent's webhook to include the header?